### PR TITLE
refactor: upgrade personalize create-trackingid.js to use aws sdk v3

### PR DESCRIPTION
**Pull request by koladilip**
## Description of the change

Upgrade personalize create-trackingid.js to use aws sdk v3. This gets rid of few deprecated dependencies but we can't get rid of all as that requires github workflows to be migrated

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: src/services/profile.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { S3Client } from '@aws-sdk/client-s3';
 import moment from 'moment';
```
Comment (sanpj2292) [inline src/services/profile.ts:4]: Are we adding this profile service as any route ?

Comment (sanpj2292) [reply src/services/profile.ts:4]: If not, shall we add a router for this ?

**Review by sanpj2292 — APPROVED** [review]

File: src/services/profile.ts (hunk 1 above)
Comment (sanpj2292) [reply src/services/profile.ts:4]: This is a non-blocking comment

Comment (koladilip) [reply src/services/profile.ts:4]: it is part of new routes @sanpj2292

Comment (sanpj2292) [reply src/services/profile.ts:4]: Oh... nice 👍🏼

File: package.json [hunk 2]
```diff
@@ -118,39 +116,32 @@
...
     "commitizen": "^4.3.0",
     "commitlint": "^17.6.3",
     "conventional-github-releaser": "^3.1.5",
     "eslint": "^8.40.0",
-    "eslint-config-airbnb": "^19.0.4",
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-config-airbnb-typescript": "^17.0.0",
     "eslint-config-prettier": "^8.8.0",
     "eslint-plugin-import": "^2.27.5",
     "eslint-plugin-json": "^3.1.0",
```
Comment (krishna2020) [inline package.json:131]: All these eslint dependencies are redundant ?

File: package.json [hunk 3]
```diff
@@ -118,39 +116,32 @@
...
     "jest-sonar": "^0.2.16",
     "jest-when": "^3.5.2",
     "lint-staged": "^13.2.2",
-    "madge": "^6.0.0",
+    "madge": "^6.1.0",
     "mocked-env": "^1.3.5",
     "node-notifier": "^10.0.1",
-    "pre-commit": "^1.2.2",
     "prettier": "^2.8.8",
+    "semver": "^7.5.3",
```
Comment (krishna2020) [inline package.json:144]: Where's `semver` library getting used ?

File: package.json (hunk 2 above)
Comment (koladilip) [reply package.json:131]: Yes checked without them also eslint is working

File: package.json (hunk 3 above)
Comment (koladilip) [reply package.json:144]: standversion has semver dependency

**Review by krishna2020 — APPROVED** [review]

**Pull request merged**